python/lancedb/query.py

```diff
@@ -30,7 +30,40 @@
 
 
 class Query(pydantic.BaseModel):
-    """A Query"""
+    """The LanceDB Query
+
+    Attributes
+    ----------
+    vector : List[float]
+        the vector to search for
+    filter : Optional[str]
+        sql filter to refine the query with, optional
+    prefilter : bool
+        if True then apply the filter before vector search
+    k : int
+        top k results to return
+    metric : str
+        the distance metric between a pair of vectors, 
+        
+        can support L2 (default), Cosine and Dot. 
+        [metric definitions][search]
+    columns : Optional[List[str]]
+        which columns to return in the results
+    nprobes : int
+        The number of probes used - optional
```

Comment:
is this list rendered in the paremeter section , or another section for explaination?

Could you check the rendered doc

Reply:
The first screenshot attached presented the rendered doc. Does it look OK?